### PR TITLE
add docstrings examples for dropout1d

### PR DESCRIPTION
**Pull request by xoiga123**
`black` auto-formatted `ifft` too

### CODE REVIEW

File: ivy/container/layers.py [hunk 1]
```diff
@@ -350,7 +350,7 @@ def dropout(
 
     @staticmethod
     def static_dropout1d(
-        x: ivy.Container,
+        x: Union[ivy.Array, ivy.NativeArray, ivy.Container],
```
Comment (hirwa-nshuti) [inline ivy/container/layers.py:353]: This instance method only accepts `ivy.Container` as input

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @xoiga123 

All looks good😊
Just a few changes and the PR get merged.

Thanks

File: ivy/container/layers.py (hunk 1 above)
Comment (xoiga123) [reply ivy/container/layers.py:353]: Sorry for the late reply. I agree that your review is correct, but I've seen many conflicting examples (i.e. elementwise) where a `ivy.Container.static_` function does accept both `ivy.Array` and `ivy.Container` as a valid type hint. In `ivy.ContainerBase.cont_multi_map_in_function`, we even assert that a container must exist in the arguments. So are the type hints wrong in these examples?

Comment (hirwa-nshuti) [reply ivy/container/layers.py:353]: Only the mentioned line should be changed as the function accepts one input as array and for the container. This means this static method can't accept the `ivy.Array` input and aswell `ivy.NativeArray` inputs.

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Check the last comment

**Review by hirwa-nshuti — APPROVED** [review]
Looks good now😊

Thanks for contributing🙂

**Pull request merged**